services/121-service/src/registration/dto/bulk-import.dto.ts

```diff
@@ -89,7 +89,10 @@ export class ImportRegistrationsDto extends BulkImportDto {
     example: fspArray.join(' | '),
   })
   @IsIn(fspArray)
-  public fspName: FinancialServiceProviderName;
+  // Should we change this to a more specific name?
```

Comment:
So for not change it programFinancialServiceProviderConfigurationName even tough it may confuse users? I'am okay with that, as it is also in the template